crates/iceberg/src/spec/manifest_list.rs

```diff
@@ -69,6 +73,25 @@ impl ManifestList {
         &self.entries
     }
 
+    /// Get the v1 schema of the manifest list entry.
+    pub(crate) fn v1_schema() -> Schema {
```

Comment:
@ZENOTME How do you feel about moving the schema definitions for the manifest entries into the `_const_fields` module?

Reply:
Seems reasonable for me. And maybe we also can let v1_schema be construct statically.